### PR TITLE
ScaFaCoS maintenance

### PR DESCRIPTION
**Pull request by jngrad**
Description of changes:
- fix a bug in the ScaFaCoS interface when calling the `get_params()` method of a ScaFaCos-based actor
- add a test for the ScaFaCos `get_params()` method
- remove unused ScaFaCoS function
- document and test FCS weights attribute
- improve testsuite readability

### CODE REVIEW

File: src/python/espressomd/scafacos.pyx [hunk 1]
```diff
@@ -45,13 +45,13 @@ IF SCAFACOS == 1:
             self._set_params_in_es_core()
 
         def valid_keys(self):
-            tmp = ["method_name", "method_params", "prefactor"]
+            tmp = self.required_keys().copy()
```
Comment (fweik) [inline src/python/espressomd/scafacos.pyx:48]: Couldn't you do `return self.required_keys().copy().add("check_neutrality")`?

Comment (fweik) [reply src/python/espressomd/scafacos.pyx:48]: I mean, wrapped in an `if` of course

Comment (jngrad) [reply src/python/espressomd/scafacos.pyx:48]: I just tried it, but the `add()` method of `set` returns None.

Comment (jngrad) [reply src/python/espressomd/scafacos.pyx:48]: and we also can't do `return self.required_keys().copy() + {"check_neutrality"}`

Comment (fweik) [reply src/python/espressomd/scafacos.pyx:48]: Not so fluent :-)

**Review by fweik — APPROVED** [review]

**Pull request merged**